### PR TITLE
Small GUI fixes, pt. 2

### PR DESCRIPTION
**Pull request by ImUrX**
* Should finally fix the status system having the unnamed tracker :3
* Added persistent advanced body parts
* add precommit hook to check if everything is well and dandy

### CODE REVIEW

**Review by Louka3000 — CHANGES_REQUESTED** [review]
~~blocking this because the pre-commit hook is fucky on GH Desktop/Windows lol~~ good now

**Review by Louka3000 — APPROVED** [review]

File: .husky/pre-commit
Comment (TheDevMinerTV) [inline .husky/pre-commit:1]: yeet this

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "prepare": "husky install && npm run update-solarxr"
   },
   "devDependencies": {
-    "husky": "^8.0.3"
+    "husky": "^8.0.3",
```
Comment (TheDevMinerTV) [inline package.json:18]: yeet this

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

**Review by TheDevMinerTV — APPROVED** [review]
Mh fine

**Review by loucass003 — APPROVED** [review]
LGTM

**Review by Eirenliel — APPROVED** [review]

**Review by Louka3000 — APPROVED** [review]

**Pull request merged**